torch/_dynamo/variables/builder.py

```diff
@@ -1176,18 +1180,34 @@ def wrap_to_fake_tensor_and_record(
             # If there is no entry for this source, add the tensor to frame state with its current static size.
             # E.g., {} -> {“x”: [2, 4]}
             curr_sizes = list(e.size())
+            log.debug("Registered static shapes %s for %s", curr_sizes, name)
```

Comment:
This one gets very noisy, so maybe we keep him debug, and make the others info? Open to bikshedding. @ezyang opine please.

Reply:
It is hard to say without actually using the logs. Keeping everything as debug to start is a safe start, assuming this doesn't trigger too much (but I don't think it does, since we debug trace every opcode in dynamo). Note that you can selectively up logging from ONLY this module with `TORCH_LOGS=+torch._dynamo.variables.builder`

Reply:
I would bikeshed the message here though; "registered" isn't very descriptive. In the log message, I think I probably want the words `frame_state` in it. Maybe `frame_state[name] = curr_sizes`. But in that case, why don't we move the log message down to where we actually do the setter, and then delete all the other branches?